### DISCUSSION TITLE
How to configure Redis?

### DISCUSSION THREAD

**Discussion by lolsokje** [beyondcode/herd-community#747] [Q&A] [answered]
Just purchased Herd Pro and I added the Redis service as it's needed for some projects at work. The way our Symfony projects are configured, it's pretty much a hardcoded requirement to have a Redis password. I figured it'd be possible to set this password somewhere, but having tried various locations for the `redis.conf` file, none of them seem to work. Neither setting `REDIS_PASSWORD=` nor `REDIS_PASSWORD=null` in the `.env` file work.

Is it at all possible to set a specific password for the Redis service?

**Comment by mpociot** [comment, marked as answer]
Hey!

Right now Redis loads the default configuration.
I thought that you could simply place a `redis.conf` file in the service directory, but it seems like redis-server ignores it unless it is explicitly specified.
I'll add support for this in the next version!

In the meantime, you can however start the service manually via your terminal:

1. Place your `redis.conf` in `/Users/shared/Herd/services/redis/7.0.0`
2. Open your terminal
3. Run `/Users/shared/Herd/services/redis/7.0.0/bin/redis-server /Users/shared/Herd/services/redis/7.0.0/redis.conf --port 6379 --daemonize no`

**Reply by lolsokje** [reply]
That works, thanks! I'll await the new release 😃
